src/sentry/static/sentry/app/views/settings/projectSourceMaps/index.tsx

```diff
@@ -15,31 +11,14 @@ type RouteParams = {
 type Props = RouteComponentProps<RouteParams, {}> & {
   organization: Organization;
   project: Project;
+  children: React.ReactNode;
 };
 
-class ProjectSourceMapsContainer extends React.Component<Props> {
-  renderNoAccess() {
-    return (
-      <PageContent>
-        <Alert type="warning">{t("You don't have access to this feature")}</Alert>
-      </PageContent>
-    );
-  }
-
-  render() {
-    const {children, project, organization} = this.props;
-
-    return (
-      <Feature
-        features={['artifacts-in-settings']}
-        organization={organization}
-        renderDisabled={this.renderNoAccess}
-      >
-        {React.isValidElement(children) &&
-          React.cloneElement(children, {organization, project})}
-      </Feature>
-    );
-  }
-}
+const ProjectSourceMapsContainer = (props: Props) => {
+  const {children, organization, project} = props;
+  return React.isValidElement(children)
+    ? React.cloneElement(children, {organization, project})
+    : null;
+};
```

Comment:
```suggestion
}
```

To go with the suggestion above.